scico/flax/inverse.py

```diff
@@ -463,3 +464,51 @@ def __call__(self, y: Array, train: bool = True) -> Array:
             x = block(alpha_ini=alpha0_i)(x, y, train)
             alpha0_i /= 2.0
         return x
+
+
+@partial(jit, static_argnums=0)
+def power_iteration(A: LinearOperator, maxiter: int = 100):
+    """Compute largest eigenvalue of a diagonalizable :class:`.LinearOperator`.
+
+    Compute largest eigenvalue of a diagonalizable
+    :class:`LinearOperator` using power iteration.
```

Comment:
Worth adding a brief note explaining why this functionality is duplicated here.

Reply:
Added a short note indicating that lax functionality is used for allowing jitting and general jax function composition.